es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -1,22 +1,17 @@
 <template>
-    <b-container class="my-450">
+    <b-container class="py-450 py-lg-800 bg-gray-50">
         <b-row>
             <b-col
                 md="7"
                 class="my-450 mx-auto text-center">
-                <h1 class="font-weight-boldest display-3">
-                    {{ errorType }}
+                <h1 class="font-weight-bolder display-3 text-gray-600">
```

Comment:
Now that we have it, can we swap out `font-weight-bolder display-3` for `post1` class?

Reply:
Done!